lib/model_patches.rb

```diff
@@ -135,4 +135,11 @@ def alert_new_response_reminders_with_alert_survey
             alias_method_chain :alert_new_response_reminders, :alert_survey
         end
     end
+
+    ContactValidator.class_eval do
+        attr_accessor :understand
+
+        validates_acceptance_of :understand,
+                                :message => N_("Please check the box to confirm your understanding")
```

Comment:
"Please confirm that you understand that WhatDoTheyKnow is not run by the government, and the WhatDoTheyKnow volunteers cannot help you with personal matters relating to government services."?